test/OpenTelemetry.Instrumentation.GrpcCore.Tests/GrpcCoreClientInterceptorTests.cs

```diff
@@ -435,8 +435,8 @@ private static async Task TestHandlerSuccess(Func<Foobar.FoobarClient, Metadata,
             Assert.Equal(originalMetadataCount, additionalMetadata.Count);
 
             // There was no parent activity, so these will be default
-            Assert.Equal(default, capturedPropagationContext.ActivityContext.TraceId);
-            Assert.Equal(default, capturedPropagationContext.ActivityContext.SpanId);
+            Assert.NotEqual(default, capturedPropagationContext.ActivityContext.TraceId);
+            Assert.NotEqual(default, capturedPropagationContext.ActivityContext.SpanId);
```

Comment:
What is the deal with this change?

Reply:
@pcwiese Looks like there is some behavior change starting DiagnosticSource version 6 onwards. In case of `netcoreapp3.1` [this](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/a50e1cdbbe869ab2204d100610067e9325a338b7/src/OpenTelemetry.Instrumentation.GrpcCore/ClientTracingInterceptor.cs#L343) was creating an activity with default context with Id in Hierarchal format. .`net6.0` onwards that behavior is changed and I see the activity is created in W3C format.

Reply:
ok

Reply:
Any chance you know how [this](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/runs/4633530604/jobs/8198843976?pr=1084#step:7:254) can be fixed?